docs/agents/wiregrid_actuator.rst

```diff
@@ -77,6 +77,10 @@ The main functions are ``insert()`` and ``eject()``.
 In the both of the functions, after the inserting/ejecting, the stopper locks the actuators again.
 However, the motor power is not turned ON or OFF during the both functions.
 
+The parameter details are here:
+- speedrate: Actuator speed rate [0.0, 5.0] (default: 0.2)
+  DO NOT use speedrate > 1.0 if el != 90 deg!!
+
```

Comment:
This doesn't render well, and also duplicates information in the auto-documented API section. My suggestion would be to drop it. If you'd like to keep it, I'd suggest a `warning::` block like I added below.

Reply:
Sorry for that. I missed that point in merging the main into my branch.
I fixed it and corrected the default parameter value for the insert/eject function.